docs/UserGuide.md

```diff
@@ -38,97 +38,204 @@
+> If TYPE is `Due Date`, the tasks and events are sorted according to due date and time of tasks and start date and time of events, earliest first.<br>
+> If no ENDTIME specified (floating tasks), sorted to bottom of list.<br>
+> If TYPE is `Priority Level`, the tasks and events are sorted according to priority level, high first.<br>
+> If TYPE is `Archived`, the Archived tasks and events are listed. They are sorted according to date the task has been marked, most recent first.<br>
+
+#### Finding all task containing any keyword in its name: `find`
+Finds tasks whose names contain any of the given keywords.<br>
+Format: `find [t/FIND_TYPE] KEYWORD [MORE_KEYWORDS]`
+
+> FIND_TYPE can be `Partial`, `Full`, `Exact`, by default FIND_TYPE is set as `Partial` <br>
+> If FIND_TYPE is `Partial`, partial keywords will be matched e.g. `task` will match ` 2103 tasks` <br>
+> If FIND_TYPE is `Full`, only full keywords will be matched e.g. `task` will not match `2103 tasks` <br>
+> If FIND_TYPE is `Exact`, the exact set of keywords will be matched e.g. `Project Meeting` will match `2103 Project Meeting` and not match `2103 Meeting` <br>
+> The search is case insensitive. e.g `task` will match `Task`<br>
+> The order of the keywords does not matter. e.g. `project meeting` will match `meeting project` <br>
+> Only the TASK_NAME and DATE are searched. <br>
+> Only full words will be matched e.g. `task` will not match `tasks` <br>
+> If FIND_TYPE is not `Exact`, tasks matching at least one keyword will be returned (i.e. `OR` search) e.g. `Project` will match `Project Meeting`<br>
 
 Examples: 
-* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
-* `add Betsy Crowe p/1234567 e/betsycrowe@gmail.com a/Newgate Prison t/criminal t/friend`
+* `find t/Full Project meeting`<br>
+  Returns any task containing names `Project` or `Meeting`, but not `meet` 
+* `find t/Exact Project meeting`<br>
+  Returns any task containing names `Project Meeting` exactly
+* `find meet CS2103`<br>
+  Returns any task containing names `meet`, or `CS2103`, including `meeting` and other word containing `meet`
+
+#### Deleting a task : `delete`
+Deletes the specified task from Savvy Tasker. Irreversible.<br>
+Format: `delete INDEX [MORE_INDEX]`
+
+> Deletes the task at the specified `INDEX`. 
+  The index refers to the index number shown in the most recent listing.<br>
+  The index **must be a positive integer** 1, 2, 3, ...
+> Allow multiple deletion
 
-#### Listing all persons : `list`
-Shows a list of all persons in the address book.<br>
-Format: `list`
+Examples: 
+* `list`<br>
+  `delete 2`<br>
+  Deletes the 2nd task in the address book.
+* `find CS1010`<br> 
+  `delete 1`<br>
+  Deletes the 1st task in the results of the `find` command.
 
-#### Finding all persons containing any keyword in their name: `find`
-Finds persons whose names contain any of the given keywords.<br>
-Format: `find KEYWORD [MORE_KEYWORDS]`
+#### Select a task : `select`
```

Comment:
select and do what? so far I did not see any operation after select.
think of using `select` as a mechanism to display all/additional details about the task, can?